### PR TITLE
1005 - BDD User Add/Edit tests.

### PR DESCRIPTION
**Pull request by NickPhura**
Code Changes/Updates
-------------------------------------------------------
Add new user and update existing user tests.
 - Add/update fuel supplier user as fuel supplier admin
 - Add/update fuel supplier user as tfrs admin
 - Add/update admin user as tfrs admin

Some general cleanup and updates.

Trello
-----------------------------------------------------
https://trello.com/c/JTtEQg23/1005-testing-user-management-0

Version
------------------------------------------------------
Rebased on latest version of 'master' as of Nov 19, 2018.

### CODE REVIEW

File: frontend/src/organizations/components/OrganizationMembersTable.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const OrganizationMembersTable = (props) => {
     minWidth: 200
   }, {
     accessor: item => item.email,
-    className: 'col-status-display',
+    className: 'col-email',
```
Comment (NickPhura) [inline frontend/src/organizations/components/OrganizationMembersTable.js]: This was just something I noticed.  The column was used for email, but had an odd 'status-display' class.

File: functional-tests/src/test/groovy/extensions/ReactJSAware.groovy [hunk 2]
```diff
@@ -1,13 +0,0 @@
-package extensions
```
Comment (NickPhura) [inline functional-tests/src/test/groovy/extensions/ReactJSAware.groovy]: This logic was moved to a Utils.groovy file to be consistent with other traits.

File: functional-tests/src/test/groovy/pages/app/NotificationsPage.groovy [hunk 3]
```diff
@@ -30,11 +32,12 @@ class NotificationsPage extends BaseAppPage {
...
-        .sort{
-          a, b ->
-            b.$('.col-credit-trade button').text().toInteger() <=> a.$('.col-credit-trade button').text().toInteger()
-        }
+      def tableRowsParent = notificationTable.$('.rt-tbody')
+      def unreadMatchingRows = tableRowsParent.children().has('.unread').has('.col-notification button', text: linkText)
+      def sortedUnreadMatchingRows = unreadMatchingRows.sort{ a, b ->
+        b.$('.col-credit-trade button').text().toInteger() <=> a.$('.col-credit-trade button').text().toInteger()
+      }
+      return sortedUnreadMatchingRows
```
Comment (NickPhura) [inline functional-tests/src/test/groovy/pages/app/NotificationsPage.groovy]: Broke the logic these 2 functions into individual steps, so it is easier to follow.

**Comment by NickPhura** [comment]
[Testing | User Management |0|](https://trello.com/c/JTtEQg23/1005-testing-user-management-0)

File: frontend/src/organizations/components/OrganizationMembersTable.js (hunk 1 above)
Comment (amichard) [reply frontend/src/organizations/components/OrganizationMembersTable.js]: good catch

**Pull request merged**